src/govee_ble/parser.py

```diff
@@ -147,6 +173,57 @@ def _process_mfr_data(
             if debug_logging:
                 _LOGGER.debug("Cleaned up packet: %s %s", mgr_id, hex(data))
 
+        if msg_length == 24 and (
+            (is_5121 := "GV5121" in local_name)
+            or (is_5122 := "GV5122" in local_name)
+            or (is_5123 := "GV5123" in local_name)
+            or (is_5125 := "GV5125" in local_name)
+            or (is_5126 := "GV5126" in local_name)
+        ):
+            sensor_type = SensorType.BUTTON
+            if is_5121:
+                self.set_device_type("H5121")
+                sensor_type = SensorType.MOTION
+            elif is_5122:
+                self.set_device_type("H5122")
+            elif is_5123:
+                self.set_device_type("H5123")
+                sensor_type = SensorType.WINDOW
+            elif is_5125:
+                self.set_device_type("H5125")
+            elif is_5126:
+                self.set_device_type("H5126")
+            b_front_of_device_id = data[:2]
+            assert b_front_of_device_id
```

Comment:
The full device ID is formed by concatenating `data[:2]` and the Bluetooth device address.

If that is not needed, I'd recommend removing the `assert` since the assertion will never fail.

Reply:
I wasn't sure what the data actually was, but since it was an unused var, I added the assert to prevent it from being removed by the linter.  But it looks like we never need it so I'll drop it.  Thanks for confirming what it is.

Reply:
removed in https://github.com/Bluetooth-Devices/govee-ble/pull/90